recipe/meta.yaml

```diff
@@ -24,13 +25,16 @@ requirements:
     - setuptools
     - wheel
   run:
-    - python >=3.6.2
-    - astroid >=2.9.0,<2.10
+    - python
```

Comment:
`python` should be `>=3.7.2` https://github.com/PyCQA/pylint/blob/566ffca72a1a08c9e7ec14383c4710d64165c592/setup.cfg#L57. Users with python 3.7.0 and 3.7.1 can face trouble using pylint 2.14.5